.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+jobs:
+  bump-version:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_PUSH_TOKEN }}
```

Comment:
This was the trick to pushing to a protected branch: https://github.com/orgs/community/discussions/25305#discussioncomment-3247409